jangaroo-maven/jangaroo-maven-plugin/src/main/java/net/jangaroo/jooc/mvnplugin/AbstractCompilerMojo.java

```diff
@@ -177,11 +182,11 @@
    * In ECMAScript, initializer values are assigned to all 'undefined' arguments.
    * In AS3, initializer values are assigned only if you call a method with less arguments.
    * An example would be
-   *     function foo(bar: string = "default"): string {
```

Comment:
If you want to reformat code, that's fine, but for review, it would help a lot if you'd do that is a separate commit!

Reply:
No need to change that now, but please do so in the future.

Reply:
It was due to auto-formatting in my file. I tried to undo it, but it failed apparently.